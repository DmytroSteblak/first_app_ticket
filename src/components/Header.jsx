import React from 'react';
import { Image, Row } from 'react-bootstrap';

const Header = () => (
    <Row>
        <Image style={{width: "120px", margin: "70px auto"}} src="/assets/airport.png" alt="icon_img"/>
    </Row>
)

export default Header;