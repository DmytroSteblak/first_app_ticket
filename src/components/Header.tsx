import React, { FC } from 'react';
import { Image, Row } from 'react-bootstrap';

import { StyledHeader } from '../styled/Header-styled';

const Header: FC = () => (
    <Row>
        <StyledHeader>
            <Image src="/assets/airport.png" alt="icon_img"/>
        </StyledHeader>
    </Row>
);

export default Header;
