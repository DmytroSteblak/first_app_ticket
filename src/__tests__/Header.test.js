import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Header from '../components/Header';

describe('Testing component <Hader/>', () => {

    it('is the header displayed', () => {
        const header = shallow(<Header />);
        expect(toJson(header)).toMatchSnapshot();
    });
});
