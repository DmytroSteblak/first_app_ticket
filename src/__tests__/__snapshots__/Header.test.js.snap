// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing component <Hader/> is the header displayed 1`] = `
<Row
  noGutters={false}
>
  <styled.div>
    <Image
      alt="icon_img"
      fluid={false}
      rounded={false}
      roundedCircle={false}
      src="/assets/airport.png"
      thumbnail={false}
    />
  </styled.div>
</Row>
`;
